how2meet/ui/components/frames.py

```diff
@@ -19,21 +19,21 @@ def frame(navtitle: str):
             .classes('items-center justify-between duration-200 p-0 px-4') \
             .style('box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1)'):
         ui.link("How2Meet", ROUTE_BASE).classes("no-underline font-bold text-teal-500 text-xl")
-        elements.label(navtitle)
         elements.button(on_click=lambda: right_drawer.toggle()).props("flat color=white icon=menu")
 
-    with ui.right_drawer().classes("gap-0 m-0 p-0 bg-neutral-900") as right_drawer:
-        with ui.row().classes("w-full gap-0"):
-            header_links = {
-                "Home": ROUTE_BASE,
-                "Events": URL_EVENTS,
-                "Settings": URL_SETTINGS,
-            }
-            for link_text, link_target in header_links.items():
-                with elements.card().on("click", lambda link_target=link_target: ui.open(link_target)).classes(
-                    "w-full no-shadow border-b-[1px] rounded-none hover:bg-gray-800"
-                ).style("cursor: pointer"):
-                    ui.link(link_text, link_target).classes("w-full text-left no-underline font-bold")
+    with ui.right_drawer()\
```

Comment:
Could probably remove the right drawer altogether. Doesn't serve much of a purpose, at least not yet

Reply:
Agreed. I almost did, but it's unobtrusive enough that it felt silly to remove it now just to add it back in when we add a feature that uses it.

Reply:
Fair enough. Though I currently can't think of a particularly good use case